spectacles/validators/data_test.py

```diff
@@ -50,6 +50,16 @@ def validate(self) -> Dict[str, Any]:
             model_name = test["model_name"]
             explore_name = test["explore_name"]
             query_url_params = test["query_url_params"]
+            file_path = test["file"].split("/", 1)[1]
```

Comment:
Is this reliable for all paths? Worried about KeyErrors happening here.

Reply:
Which bit are you worried about? I'm pretty confident the `file` key is on every test returned and it is always prefixed with `projectname/`.

Reply:
A completely unfounded concern that the path format might differ in different setups. I would be satisfied if we catch an `IndexError` when we do the `split` + indexing and return a sensible error message about the path having an unexpected format.